### ISSUE TITLE
Internal Error: database.lua:95: No such tableName in setView!

### ISSUE THREAD

**Issue by jbmorley** [inseven/opolua#328]
## Description

- Biklog5 (https://archive.org/download/3-libjune-05/3LIBJUNE05.iso/library%2Fepocmisc%2Fbiklog5%2Fbiklg532.sis)

## Details

```
database.lua:95: No such tableName in setView!
INI\INI_OPEN%:00000164: B4 [Open] logName=8 fields=NAME$, STRING$, INT%, LONG&, FLOAT
INI\GETLASTUSEDFILE$:000006EC: 53 [RunProcedure] 0x0012 (name="INI_OPEN%" nargs=1)
BIKLOG5\MAIN2:00000A2D: 53 [RunProcedure] 0x06B2 (name="GETLASTUSEDFILE$" nargs=1)
TOOLBAR\TBARLINK: [Lua] modules/toolbar.lua:66
BIKLOG5\MAIN1:000000F7: 53 [RunProcedure] 0x0012 (name="TBARLINK" nargs=1)
IQGRID\GRID_LINK%:0000049E: 6B [CallProcByStringExpr] nargs=0 type=�
BIKLOG5\START:00000098: 53 [RunProcedure] 0x0022 (name="GRID_LINK%" nargs=1)
stack traceback:
	[C]: in function 'assert'
	database.lua:95: in method 'setView'
	runtime.lua:639: in method 'openDb'
	ops.lua:1587: in function 'ops.Open'
	runtime.lua:859: in function <runtime.lua:844>
	[C]: in function 'xpcall'
	runtime.lua:978: in method 'pcallProc'
	runtime.lua:1019: in method 'callProc'
	modules/toolbar.lua:66: in field 'fn'
	runtime.lua:310: in method 'pushNewFrame'
	ops.lua:998: in function 'ops.RunProcedure'
	runtime.lua:859: in function <runtime.lua:844>
	[C]: in function 'xpcall'
	runtime.lua:978: in method 'pcallProc'
	runtime.lua:1242: in function 'runtime.runOpo'
```

**Comment by tomsci** [comment]
`"C:\System\Apps\biklog5\biklog5.ini" SELECT name,string,integer,long,float FROM deftable`

The quoting confused the checks, this should've errored with `unimplemented("database.parseTableSpec")` because we don't support `SELECT ... FROM` (yet).

**Comment by tomsci** [comment]
Also looks like there's a nonprintable characted in the stacktrace of the CallProcByStringExpr call, sigh...

**Comment by tomsci** [comment]
Fixed in #332.
